### PR TITLE
[Feature][#2] Format csv data

### PR DESCRIPTION
**Pull request by qiuqili**
https://trello.com/c/dQpeXCys

CSV

- map csv headers, values to lowercase
- replace csv headers with readable string
- clean up some code

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -31,60 +31,38 @@ if (process.env.NODE_ENV !== "test") {
...
+  .pipe(
+    csvParser({
+      headers: [
+        "cohort",
+        "shortlist",
+        "applicant_id",
+        "gender",
+        "pronouns",
+        "minority",
+        "hear_from",
```
Comment (caropi) [inline src/index.js]: Could we change this instead "bridge_referral_from" just for a bit more clarity?

File: src/index.js [hunk 2]
```diff
@@ -31,60 +31,38 @@ if (process.env.NODE_ENV !== "test") {
...
+        "applied_cohort",
+        "employment",
```
Comment (caropi) [inline src/index.js]: Could this change to "employment_status" just to define that we're looking at the current employment and not the desired outcome from bridge? 

It makes sense when we're looking at the data as a whole, but if anyone besides us are trying to reach the specific endpoint they might be confused about the granular differences

File: src/index.js [hunk 3]
```diff
@@ -31,60 +31,38 @@ if (process.env.NODE_ENV !== "test") {
...
+        "newjob",
```
Comment (caropi) [inline src/index.js]: Can this be changed to "seeking_job_post_bridge" I know it's a tiny bit verbose, but it's clear what it is.

Comment (qiuqili) [reply src/index.js]: I try to keep it maximum two words ,how about I changed to three words? I think four is a bit too long

Comment (caropi) [reply src/index.js]: What three words are you suggesting?

Comment (qiuqili) [reply src/index.js]: seeking_job_after? or I can go with yours if you really like this one

Comment (caropi) [reply src/index.js]: That works for me! You'll just need to push the changes to the headers before I can approve :)

**Pull request merged**